packages/aws-cdk-lib/core/lib/custom-resource.ts

```diff
@@ -54,6 +55,8 @@ export interface CustomResourceProps {
    *   serviceToken: myTopic.topicArn,
    * });
    * ```
+   *
+   * serviceToken is passed to properties with key "ServiceToken"
```

Comment:
```suggestion
   * Maps to [ServiceToken](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cloudformation-customresource.html#cfn-cloudformation-customresource-servicetoken) properties for the `AWS::CloudFormation::CustomResource` resource
```

This is a nit but would the above reads better? I think the links would help. Same goes to the `serviceTimeout` code comment (line 68 and 69).

Reply:
I agree yeah, help with clarity

Reply:
Added